#Service to start execution of joint waypoints
---
